src/types/hermesProfileInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+import { SharedEventProperties } from './chromeEventInterfaces';
+
+/**
+ * Each item in the stackFrames object of the hermes profile
+ */
+export interface HermesStackFrame {
```

Comment:
Define an interface for an individual stackFrame, so the typedef for stackFrames in the hermes profile would be ```{[key in string]: HermesStackFrame}```